### PR TITLE
Add pyfish recipe

### PR DESCRIPTION
**Pull request by tlkaufmann**
Adding pyfish which is a plotting utility for creating Fish / Muller plots.

Recipe created using `conda skeleton pypi pyfish`

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by tlkaufmann** [comment]
@BiocondaBot please add label

File: recipes/pyfish/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+requirements:
+  host:
+    - matplotlib-base >=3.0
+    - numpy >=1.14
+    - pandas >=1.0
+    - pip
+    - pytest >=3.0
+    - python
+    - scipy >=1.0
```
Comment (marcelm) [inline recipes/pyfish/meta.yaml:27]: You only need to list packages necessary for `pip install`. That should be only `python` and `pip`

File: recipes/pyfish/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  run:
+    - matplotlib-base >=3.0
+    - numpy >=1.14
+    - pandas >=1.0
+    - pytest >=3.0
```
Comment (marcelm) [inline recipes/pyfish/meta.yaml:32]: Why is pytest needed? That is usually a development-only requirement.

File: recipes/pyfish/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    - python
```
Comment (marcelm) [inline recipes/pyfish/meta.yaml:33]: The `setup.py` has `python_requires='>=3.8'`, so you should also write `python >=3.8` here.

**Review by marcelm — APPROVED** [review]
Looks good!

**Comment by marcelm** [comment]
You’re welcome!

**Pull request merged**